### PR TITLE
Support Resize `nearest` mode

### PR DESCRIPTION
**Pull request by ishiy1993**
- Add `Resize2D` to nnoir
- Convert`Resize` in onnx to `Resize2D` in nnoir
- Remain `Bilinear2D` in nnoir for compatibility

### CODE REVIEW

**Comment by koiking213** [comment]
Could you add a `linear` test for `nnoir/test/test_run.py`?

File: nnoir/nnoir/functions/resize_2d.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    def run(self, x):
+        out_h = self.params["size"][0]
+        out_w = self.params["size"][1]
+        modes = self.params["interpolation_mode"].decode("utf-8").split("-")
+        if modes[0] == "linear":
+            return _run_linear(x, out_h, out_w, self.params["coordinate_transformation_mode"])
+        elif modes[0] == "nearest":
+            return _run_nearest(x, out_h, out_w, self.params["coordinate_transformation_mode"], modes[1])
+        else:
+            raise Exception("unkonw Resize mode")
```
[the remark this answers is not in the document]
Comment (ishiy1993) [reply nnoir/nnoir/functions/resize_2d.py]: fixed :pray:

**Comment by ishiy1993** [comment]
@koiking213 Add `Resize2D_linear.nnoir`. Please review again :pray:

**Review by koiking213 — APPROVED** [review]
LGTM

**Pull request merged**